Microsoft.Toolkit.Uwp.UI/Cache/CacheBase.cs

```diff
@@ -79,17 +89,58 @@ public int MaxMemoryCacheCount
         }
 
         /// <summary>
+        /// Gets instance of <see cref="HttpClient"/>
+        /// </summary>
+        protected HttpClient HttpClient
+        {
+            get
+            {
+                if (_httpClient != null)
+                {
+                    return _httpClient;
+                }
+
+                var messageHandler = new HttpClientHandler() { MaxConnectionsPerServer = 10 };
```

Comment:
@cbarkerms they can specify their own HttpMessageHandler which is then used when instantiating the httpClient.. we can do one or the other..